docs/user/app_feature_intended.md

```diff
@@ -95,7 +96,7 @@ In order to generate the intended configurations at least two repositories are n
 ### Intended Repository Matching Rule
 
 !!! note
-    Only use a Intended Repository Matching Rule if you have **more than one** intended repository. It is **not needed"" if you only have one repository. The operator is expected to ensure that every device results in a successful matching rule (or that device will fail to render a config).
+Only use a Intended Repository Matching Rule if you have **more than one** intended repository. It is \*\*not needed"" if you only have one repository. The operator is expected to ensure that every device results in a successful matching rule (or that device will fail to render a config).
```

Comment:
please revert

Reply:
yes, sure. I will do this with the last update of the docs, as I guess more auto style will happen.